pages/governance/proposal/[proposalId].governance.tsx

```diff
@@ -440,21 +441,71 @@ export default function ProposalPage({
                   >
                     <Box sx={{ textAlign: 'right' }}>
                       <Typography>
-                        ~ {dayjs.unix(proposal.startTimestamp).format('DD MMM YYYY, hh:mm a')}
+                        {dayjs.unix(proposal.startTimestamp).format('D MMM YYYY, HH:mm UTC Z')}
                       </Typography>
                       <Typography variant="caption" color="text.muted">
                         {proposal.startBlock}
                       </Typography>
                     </Box>
                   </Row>
+                  {dayjs() > dayjs.unix(proposal.expirationTimestamp) && (
```

Comment:
I think this should be assigned to a variable for easier understanding of what this is.  Something like:

```js
const proposalHasExpired = dayjs() > dayjs.unix(proposal.expirationTimestamp)
```

Then it's very clear as to when and why this block of UI gets rendered.

Reply:
done